src/cli/scenarios.py

```diff
@@ -49,6 +50,26 @@ def run(scenario, network, additional_args):
     print(rpc_call("scenarios_run", params))
 
 
+@scenarios.command(context_settings={"ignore_unknown_options": True})
+@click.argument("scenario_path", type=str)
+@click.argument("additional_args", nargs=-1, type=click.UNPROCESSED)
+@click.option("--network", default="warnet", show_default=True)
+def run_file(scenario_path, network, additional_args):
+    """
+    Run <scenario_path> from the Warnet Test Framework on <--network> with optional arguments
```

Comment:
nit: not sure if `scenario_path` is enough of a distinction to let a user know how this command is different from the command above. also have a slight aversion to `_` in the command verb names.

Reply:
Agree with the nit. Hopefully we can have a chat about how we want scenarios to be in the future and make it cleaner / clearer. I just wanted something now to unblock testing.